apps/frontend/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const { composePlugins, withNx } = require('@nrwl/webpack');
+const { withReact } = require('@nrwl/react');
+const TerserJSPlugin = require('terser-webpack-plugin');
+
+// Nx plugins for webpack.
+module.exports = composePlugins(withNx(), withReact(), (config) => {
+  // Update the webpack config as needed here.
+  // e.g. `config.plugins.push(new MyPlugin())`
+
+  // https://github.com/nrwl/nx/issues/10017#issuecomment-1405429851
```

Comment:
Can you leave a comment as to why we added this
```suggestion
  // https://github.com/nrwl/nx/issues/10017#issuecomment-1405429851
  // Fix to web workers not working
```